marketing-checklist.md

```diff
@@ -167,7 +167,10 @@ This section pairs nicely with the [Post-Launch section of the Sales Checklist](
 - [ ] Create an online course or guide around your product/industry (be sure to [check out this list of course management tools](https://www.sideprojectchecklist.com/2017/course-creation-tools)).
 - [ ] Run an [influencer marketing campaign](https://www.reddit.com/r/startups/comments/7aap5u/influencer_marketing_summary_of_12_articles/).
 - [ ] Run a link-building campaign.
-- [ ] Share the story of how you started your business on [Starter Story](https://starterstory.com) - it's free and you get a do-follow backlink.
+
+  ##### Share the story of how you started your business for free and get a do-follow backlink
+  - [ ] [Starter Story](https://starterstory.com)
+  - [ ] [Oyestartups](https://oyestartups.com)
```

Comment:
Are there any other sources we can include? I like having 3 or more options for each section if we can.

Maybe Indie Hackers?

Or one of these: https://startup.info/ https://www.learnthestartup.com/ https://www.balancethegrind.com.au/

Reply:
IndieHacker and Learn the startup will be a good addition